### PR TITLE
fix: update icon generate script

### PR DESCRIPTION
**Pull request by Scrumplex**
Some Windows users were having issues with low-res icons.

I feel like 5MB for a .ico is a bit too big. Any ideas?

### CODE REVIEW

**Comment by txtsd** [comment]
5MB over network is huge. It's fine for a local file.

**Comment by ZekeZDev** [comment]
That is very big for an ico. The current website one is 14kb

**Comment by Scrumplex** [comment]
Apparently Windows ICOs support 256x256 at most :|

File: program_info/genicons.sh [hunk 1]
```diff
@@ -1,39 +1,73 @@
-#/bin/bash
+#!/bin/bash
```
Comment (txtsd) [inline program_info/genicons.sh:1]: `#!/usr/bin/env bash` might be safer to use.
I do all my scripts like this.

**Review by txtsd — APPROVED** [review]
But it's okay otherwise

File: program_info/genicons.sh (hunk 1 above)
Comment (Scrumplex) [reply program_info/genicons.sh:1]: As per POSIX `/bin/bash` is basically guaranteed

**Comment by Scrumplex** [comment]
bump

**Review by flowln — APPROVED** [review]

**Pull request merged**